tools/check-tracking-number-and-date.mjs

```diff
@@ -0,0 +1,48 @@
+import { parse as parseTOML } from 'toml';
+import { readFileSync } from 'fs';
+import { campaignInfoToCampaignConfig } from '../webpack/convert_info_to_type.js';
+
+const campaignConfig = parseTOML( readFileSync( 'campaign_info.toml', 'utf-8' ) );
+const configObject = campaignInfoToCampaignConfig( campaignConfig );
+
+for ( const configEntry of Object.entries( configObject ) ) {
+	const configValues = configEntry[ 1 ];
+
+	// check test number
+	let testNumber = '';
+	const testNumberMatchResult = configValues.campaign.match( /_\d{2}/ );
+	if ( testNumberMatchResult === null ) {
+		console.log( 'Cannot parse test number ( "_dd" format) for ' + configValues.campaign );
+		break;
+	}
+	testNumber = testNumberMatchResult[ 0 ];
+	const testNumberSearchString = testNumber.slice( 1 ) + '-';
+	if ( !configValues.tracking.includes( testNumberSearchString ) ||
+		!configValues.banners.ctrl.tracking.includes( testNumberSearchString ) ||
+		!configValues.banners.var.tracking.includes( testNumberSearchString ) ) {
+		console.log(
+			`\x1b[48;2;139;0;0m\x1b[37m Faulty tracking config (test number) for: \x1b[0m\x1b[48;2;139;0;0m\x1b[37m${configValues.campaign} \x1b[0m`
+		);
+		break;
```

Comment:
I want to fail as early as possible because if the "parsing steps" already don't succeed, the "comparison steps" also should not be executed either.
So I would like to `exit(1)` within the loop even